public/scripts/jquery.deserialize.js

```diff
@@ -1,12 +1,43 @@
-jQuery.fn.deserializeObject = function(hash) {
-  for (name in hash) {
-    var elem = $("[name='" + name + "']", this);
-    if (elem.is('select[multiple], :radio, :checkbox')) {
-      elem.val([hash[name]]);
-    }
-    else{
-      elem.val(hash[name]);
-    }
-  }
-  return $(this);
-};
+jQuery.fn.deserializeObject = function(hash){
+
+	t = this;
```

Comment:
Why do you define the variables as global variables?

Reply:
My bad, will fix.